src/main/java/racing/Name.java

```diff
@@ -0,0 +1,36 @@
+package racing;
+
+import java.util.Objects;
+
+class Name {
+    private String name;
```

Comment:
```suggestion
    private final String name;
```

가급적 부수효과가 적은 불변 객체로 만들어 주세요.